website/content/project/presentation.md

```diff
@@ -1,22 +1,30 @@
 # Final Presentations
 
-{{% warning %}}
-**NOTE**: These are the requirements for the 2019 edition of CMSC 22000. They have not yet been updated for 2020!
-{{% /warning %}}
+The final component of this course will be a series of final presentations, where each team will present the work they have done throughout the quarter. This presentation will be divided into two components:
 
-The final component of this course will be a series of final presentations, where each team will present the work they have done throughout the quarter. The presentations will take place on Wednesday, June 12, from 1:30pm to 3:30pm, in Ryerson 277. Each team will have 15 minutes to present (with an additional 5 minutes for questions).
+- A pre-recorded presentation that must be submitted by **Tuesday, June 9, at 12pm CDT**.
+- A 5-10 minute Q&A session with the instructor and TAs during the "final exam" slot for the class: **Tuesday, June 9, from 4pm to 6pm CDT**
+
+Once all presentations have been submitted, they will be shared with the entire class, so that other teams can learn about the work that other teams have done (similar to how, in live presentations, you would be able to attend the presentations for all the teams).
```

Comment:
Minor point about the language: "other" is repeated here, so maybe the first "other teams" should be changed to "teams" instead?

Reply:
Sounds good, I've tweaked the wording.